README.md

```
@@ -199,6 +199,32 @@ q := chizuql.New().
 sql, args := q.Build()
 ```
 
+### Filtro com NOT IN + CastAsAny (usando Or)
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)